umake/frameworks/ide.py

```diff
@@ -473,3 +474,83 @@ def post_install(self):
                                                      categories=categories))
         if not self.was_in_arduino_group:
             UI.delayed_display(DisplayMessage(_("You need to logout and login again for your installation to work")))
+            
+class BaseNetBeans(umake.frameworks.baseinstaller.BaseInstaller):
+    """The base for all Netbeans installers."""
+    
+    BASE_URL = "http://download.netbeans.org/netbeans/8.0.2/final/"
```

Comment:
I would prefer we do not encode any version here, as the goal of Ubuntu Make is to always be able to download latest versions of any IDE, and back that with tests. Is there a way to go to a more generic download page, get the version (or a redirect, that is transparently supported by the framework) which will lead to the desired page?